tests/cypress/page-object/site.home.page.ts

```diff
@@ -1,20 +1,26 @@
 import { BasePage } from './base.page'
 import { editPage } from './edit.page'
+import { workflowDashboard } from './worflow.dashboard.page'
 
 class SiteHomePage extends BasePage {
     elements = {
         iframePageComposerFrame: 'iframe[id="page-composer-frame"]',
         iframeNestedSrcEditFrame: 'iframe[src*="editframe"]',
-
         divRoleRow: 'div[role="row"]',
         imgVirtualSite: 'img[src*="jnt_virtualsite"]',
-
-        textEdit: 'Edit',
+        publishSiteinAllLang: "[class*='publishsiteinalllanguages']",
+        editSite: "[class*='editcontentroot']",
+        cacheButton: '.edit-menu-cache',
+        flushAll: "[class*='flushall']",
     }
 
-    goTo(siteHomeUrl: string) {
+    goTo(siteHomeUrl = '/jahia/page-composer/default/en/sites/digitall/home.html') {
```

Comment:
I would prefer URL as input for now (no default)
Can have sample in the comments

Reply:
sure, works either way for me